x-pack/plugins/graph/server/index.ts

```diff
@@ -16,6 +16,7 @@ export const config: PluginConfigDescriptor<ConfigSchema> = {
   exposeToBrowser: {
     canEditDrillDownUrls: true,
     savePolicy: true,
+    enabled: true,
```

Comment:
nit: I don't think it's worth exposing it to the browser.

The `enabled` config is handled by core, completely disabling (not loading) the plugin.

Reply:
Done!